internal/pkg/cli/app_status.go

```diff
@@ -0,0 +1,446 @@
+	)
+	if err != nil {
+		return fmt.Errorf("select deployed applications for project %s: %w", o.ProjectName(), err)
+	}
+	o.appName = appEnvs[appEnvName].appName
+	o.envName = appEnvs[appEnvName].envName
+
+	return nil
+}
+
+func (o *appStatusOpts) retrieveAllAppNames() ([]string, error) {
+	apps, err := o.storeSvc.ListApplications(o.ProjectName())
+	if err != nil {
+		return nil, fmt.Errorf("list applications for project %s: %w", o.ProjectName(), err)
+	}
+	appNames := make([]string, len(apps))
+	for ind, app := range apps {
+		appNames[ind] = app.Name
+	}
+
+	return appNames, nil
+}
+
+func (o *appStatusOpts) configSvc() error {
+	if err := o.initDescriber(o, o.appName); err != nil {
+		return err
+	}
+	env, err := o.storeSvc.GetEnvironment(o.ProjectName(), o.envName)
+	if err != nil {
+		return fmt.Errorf("get environment: %w", err)
+	}
+	if err := o.initcwSvc(o, env); err != nil {
+		return err
+	}
+	return o.initecsSvc(o, env)
+}
+
+// JSONString returns the stringified webAppStatus struct with json format.
+func (w *webAppStatus) JSONString() (string, error) {
+	b, err := json.Marshal(w)
+	if err != nil {
+		return "", fmt.Errorf("marshal applications: %w", err)
+	}
+	return fmt.Sprintf("%s\n", b), nil
+}
+
+// HumanString returns the stringified webAppStatus struct with human readable format.
+func (w *webAppStatus) HumanString() string {
+	var b bytes.Buffer
+	writer := tabwriter.NewWriter(&b, minCellWidth, tabWidth, cellPaddingWidth, paddingChar, noAdditionalFormatting)
+	fmt.Fprintf(writer, color.Bold.Sprint("Service Status\n\n"))
+	writer.Flush()
+	fmt.Fprintf(writer, "  %s\t%s\n", "Status", w.Service.Status)
+	fmt.Fprintf(writer, "  %s\t%v\n", "DesiredCount", w.Service.DesiredCount)
+	fmt.Fprintf(writer, "  %s\t%v\n", "RunningCount", w.Service.RunningCount)
+	fmt.Fprintf(writer, color.Bold.Sprint("\nTask Status\n\n"))
+	writer.Flush()
+	fmt.Fprintf(writer, "  %s\t%s\t%s\t%s\t%s\t%s\n", "ID", "ImageDigest", "LastStatus", "DesiredStatus", "StartedAt", "StoppedAt")
+	for _, task := range w.Tasks {
+		var digest []string
+		for _, image := range task.Images {
+			digest = append(digest, image.Digest[:shortImageDigestLength])
+		}
+		startedSince := since(task.StartedAt)
+		stoppedSince := "-"
+		if task.StoppedAt != 0 {
+			stoppedSinceDuration := since(task.StoppedAt)
+			stoppedSince = stoppedSinceDuration.String()
+		}
+		fmt.Fprintf(writer, "  %s\t%s\t%s\t%s\t%s\t%s\n", task.ID[:shortTaskIDLength], strings.Join(digest, ","), task.LastStatus, task.DesiredStatus, startedSince.String(), stoppedSince)
+	}
+	fmt.Fprintf(writer, color.Bold.Sprint("\nMetrics\n\n"))
+	writer.Flush()
+	fmt.Fprintf(writer, "  %s\t%s\t%s\t%s\n", "Name", "Health", "UpdatedTimes", "Reason")
+	for _, metric := range w.Metrics {
+		updatedTimeSince := since(metric.UpdatedTimes)
+		fmt.Fprintf(writer, "  %s\t%s\t%s\t%s\n", metric.Name, metric.Status, updatedTimeSince.String(), metric.Reason)
+	}
+	writer.Flush()
+	return b.String()
+}
+
+type duration time.Duration
```

Comment:
I think it's worthwhile for us to import https://github.com/dustin/go-humanize#times less likely that we'll have a bug, and there are other cool stuff we can use :D

Reply:
lol didn't know this package exists. Great idea!